server/utils/bot/bot.go

```diff
@@ -14,10 +15,12 @@ type Bot interface {
 	Poster
 	Logger
 	Admin
+	Flower
```

Comment:
When I read this it makes me think of the plant flower :rose:. Not sure what else to name this though

Reply:
Exactly the same feeling here. But if something that posts is a poster, and something that logs is a logger, something that flows...